frontend/components/AddHostsModal/AddHostsModal.tsx

```diff
@@ -58,19 +65,20 @@ const AddHostsModal = ({
       );
     }
 
-    // TODO: Currently, prepacked installers in Fleet Sandbox use the global enroll secret,
-    // and Fleet Sandbox runs Fleet Free so the currentTeam check here is an
-    // additional precaution/reminder to revisit this in connection with future changes.
-    // See https://github.com/fleetdm/fleet/issues/4970#issuecomment-1187679407.
-    return isSandboxMode && !isAnyTeamSelected ? (
+    return shouldRenderDownloadInstallersContent ? (
       <DownloadInstallers onCancel={onCancel} enrollSecret={enrollSecret} />
     ) : (
       <PlatformWrapper onCancel={onCancel} enrollSecret={enrollSecret} />
     );
   };
 
   return (
-    <Modal onExit={onCancel} title={"Add hosts"} className={baseClass}>
+    <Modal
+      onExit={onCancel}
+      title={"Add hosts"}
+      className={baseClass}
+      width={shouldRenderDownloadInstallersContent ? "large" : "medium"}
```

Comment:
With my next PR, it's always large because the ChromeOS tab needs more space, but no worries, it'll be fixed there